src/wlapplication_messages.cc

```diff
@@ -19,177 +19,246 @@
+	  {"", "rolling_autosave", _("n"), _("Use n files for rolling autosaves"), true},
+	  {"", "nozip", "", _("Do not save files as binary zip archives"), false},
+	  {"", "display_replay_filenames", _("[true*|false]"), _("Show filenames in replay screen"),
+		true},
+	  {"", "editor_player_menu_warn_too_many_players", _("[true*|false]"),
+		_("Enable verbose debug messages"), true},
+	  /// Game options
+	  {"", "auto_roadbuild_mode", _("[true*|false]"), _("Start building road after placing a flag"),
+		true},
+	  {"", "display_flags", _("[...]"), _("Display flags to set for new games"), true},
+#if 0  // TODO(matthiakl): Re-add training wheels code after v1.0
+	{"",
+	 "training_wheels",
+	 _("[true*|false]"),
+	 "",
+	 true
+	},
+#endif
+	  {"", "edge_scrolling",
+		/** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+		/** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
+		/** TRANSLATORS: * marks the default value */
+		_("[true|false*]"), _("Scroll when the mouse cursor is near the screen edge"), true},
+	  {"", "numpad_diagonalscrolling", _("[true|false*]"),
+		_("Allow diagonal scrolling with the numeric keypad"), true},
+	  {"", "game_clock", _("[true|false*]"), _("Display system time in the info panel"), true},
+	  {"", "ctrl_zoom", _("[true|false*]"), _("Zoom only when Ctrl is pressed"), true},
+	  {"", "single_watchwin", _("[true|false*]"), _("Use single watchwindow mode"), true},
+	  {"", "transparent_chat", _("[true*|false]"),
+		_("Show in-game chat with transparent background"), true},
+	  {"", "toolbar_pos", _("[...]"), _("Sets the toolbar location and mode"), true},
+	  /// Networking
+	  {_("Networking:"), "write_syncstreams",
+		/** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+		/** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
+		/** TRANSLATORS: * marks the default value */
+		_("[true*|false]"),
+		/** TRANSLATORS: A syncstream is a synchronization stream. Syncstreams are used in multiplayer
+		 */
+		/** TRANSLATORS: games to make sure that there is no mismatch between the players. */
+		_("Create syncstream dump files to help debug network games"), false},
+	  {"", "metaserver", _("URI"), _("Connect to a different metaserver for internet gaming"),
+		false},
+	  {"", "metaserverport", _("n"), _("Port number n of the metaserver for internet gaming"),
+		false},
+	  {"", "servername", _("[...]"), _("The name of the last hosted game"), true},
+	  {"", "realname", _("[...]"), _("The nickname used for LAN and online games"), true},
+	  {"", "nickname", _("[...]"), _("Name of map author"), true},
+	  {"", "addon_server", _("URI"),
+		_("Connect to a different github repository\nand branch from the add-ons manager"), false},
+	  /// Interface options
+	  {_("Graphic options:"), "fullscreen", _("[true|false*]"),
+		_("Whether to use the whole display for the\ngame screen"), false},
```

Comment:
I think that those linebreaks in the middle of strings should also go away. Translators are still forced to decide where to place them sensibly. It would be better to insert them automatically after a certain text width like in #4805.